client/dashboard/sites/logs-activity/dataviews/actions.tsx

```diff
@@ -28,6 +30,13 @@ export function useActivityActions( {
 			isEligible: ( item ) => item.is_rewindable,
 			callback: async ( items ) => {
 				const [ item ] = items;
+				if ( item ) {
+					recordTracksEvent( 'calypso_dashboard_sites_activity_logs_restore_point_click', {
```

Comment:
```suggestion
					recordTracksEvent( 'calypso_dashboard_sites_logs_activity_restore_point_click', {
```

IMO this is better - hosting dashboard `feature` folder is a combination of path/namespacing and feature name, rather than just the feature name. WDYT?

Reply:
I like this. Implemented in https://github.com/Automattic/wp-calypso/pull/106215/commits/e1d6ac49bc28c730ebb7e1778ecbfb52bd28c423

Reply:
Everything is now prefixed with `calypso_dashboard_sites_logs_activity_`